src/legacy/ManifestTranslator.ts

```diff
@@ -32,17 +32,15 @@ import { OpportunityTabExtension } from '../manifest/extensions/tabs/types/Oppor
 import { ProspectActionTabExtension } from '../manifest/extensions/tabs/types/ProspectActionTabExtension';
 import { ProspectTabExtension } from '../manifest/extensions/tabs/types/ProspectTabExtension';
 import { ReportsTabExtension } from '../manifest/extensions/tabs/types/ReportsTabExtension';
+import { ManifestApiClient } from '../manifest/ManifestApiClient';
 import { ManifestAuthor } from '../manifest/ManifestAuthor';
 import { ManifestStore } from '../manifest/ManifestStore';
 import { StoreType } from '../manifest/store/StoreType';
 import { Locale } from '../sdk/Locale';
 import { ManifestV1 } from './ManifestV1';
 
 export class ManifestTranslator {
-  public static getAddonManifest(
-    app: Application,
-    ext: TabExtension | ShellExtension
-  ): ManifestV1 | null {
+  public static getAddonManifest(app: Application, ext: TabExtension | ShellExtension): ManifestV1 | null {
```

Comment:
It allows providing only `ext: TabExtension | ShellExtension` but in code below we're checking a lot more allowed `ext.type`s

Reply:
I see only `ShellExtensionType` and `TabExtensionType` checks.

Reply:
You are right.